src/__tests__/CharacterCard.test.tsx

```diff
@@ -29,4 +37,20 @@ describe('CharacterCard component', () => {
     expect(gender).toBeInTheDocument();
     expect(species).toBeInTheDocument();
   });
+
+  it('adds and removes character from selectedItems on heart icon click', () => {
+    render(<MockCharacterCard />);
+    const icon = screen.getByTestId('icon');
```

Comment:
It's good practice to use `getTestId` as a last resort. It looks like this icon acts like a button, so we can add some accessibility practice in code and use selectors like `getByRole` or `getByLabel` (or something like that)